app/src/main/res/drawable/ic_delete_24dp.xml

```diff
@@ -0,0 +1,21 @@
+<!--
```

Comment:
NAK.
ic_delete_24dp.xml which `24dp` .

Reply:
Sorry, I'm not sure I understand.

Reply:
sorry,  🤕 my bad.
I mean the file name is strange that it is with "24dp" .

Reply:
I think it's OK.  The filter icon also has a "24dp" suffix.